odl/diagnostics/operator.py

```diff
@@ -96,7 +96,7 @@ def norm(self):
         """
         self.log('\n== Calculating operator norm ==\n')
 
-        operator_norm = max(power_method_opnorm(self.operator, niter=2,
+        operator_norm = max(power_method_opnorm(self.operator, maxiter=2,
```

Comment:
Good. We should always use `maxiter` if the iteration can stop early, and `niter` if the iteration number is fixed.